### DISCUSSION TITLE
Mailer implementation like rails mailer

### DISCUSSION THREAD

**Discussion by Jarrodsz** [remix-run/remix#4379] [Proposals]
Could be extended with transporters like
Formspree
Other cloud form tools
Local loke nodemailer

**Comment by sergiodxa** [comment]
It would be great if you can provider more information on how this could work, specially the API you think this should have, imagine this exists, how would you like to use it?

**Reply by Jarrodsz** [reply]
https://medium.com/nerd-for-tech/implementing-action-mailer-ruby-on-rails-1766f59c6f

This outlines the esse of use
